packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.scss

```diff
@@ -77,6 +77,7 @@ $DropDown-item-height: 32px;
       @include highContrastItemAndTitleState();
     }
     .caretDown {
+      color: HighlightText;
```

Comment:
Have you verified that this doesn't affect the caret's visual representation when not viewed in high-contrast mode?

Reply:
Followed up offline--the change doesn't affect the non-HC representation. Looks good!